contracts/series/SeriesController.sol

```diff
@@ -51,10 +53,14 @@ contract SeriesController is
 
     /// @notice Check if the msg.sender is the privileged DEFAULT_ADMIN_ROLE holder
     modifier onlyOwner() {
-        require(
-            hasRole(DEFAULT_ADMIN_ROLE, msg.sender),
-            "SeriesController: Caller is not the owner"
-        );
+        require(hasRole(DEFAULT_ADMIN_ROLE, msg.sender), "!admin");
+
+        _;
+    }
+
+    /// @notice Check if the msg.sender is the privileged SERIES_DEPLOYER_ROLE holder
+    modifier onlySeriesDeployer() {
+        require(hasRole(SERIES_DEPLOYER_ROLE, msg.sender), "!deployer");
```

Comment:
Should we include the owner role here as well? That way a series can be created manually by the owner multisig as well.

Reply:
Yep, that change looks good.